stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/state/messagelist/MessageFocusState.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.getstream.chat.android.compose.state.messages.list
+package io.getstream.chat.android.common.state.messagelist
```

Comment:
Small question about the naming. I prefer `messsagelist` as well, but we have `message.list` everywhere else. What do you think?